src/go/k8s/helm-chart/charts/redpanda-operator/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: redpanda
+description: Redpanda operator chart
```

Comment:
"Redpanda is the real-time engine for modern apps." as in the original chart?

Reply:
The helm chart doesn't deploy redpanda. The description should be scoped to the exact purpose of the helm chart. The README.md can have the catch phrase.

I will only change the name of the Chart `redpanda` -> `redpanda-operator`

Examples:

```
# helm search hub operator -o yaml
- app_version: ""
  description: secureCodeBox Operator to automate the execution of security scans
    on kubernetes
  url: https://hub.helm.sh/charts/seccurecodebox/operator
  version: 2.5.2
- app_version: 3.0.0.5-7e7a22e
  description: Manage and vizualize Kafka clusters with CMAK (prev. Kafka Manager).
  url: https://hub.helm.sh/charts/cmak-operator/cmak-operator
  version: 1.4.2
- app_version: 1.4.1
  description: 'A Helm chart for deploying the Elastic Cloud on Kubernetes (ECK) operator:
    the official Kubernetes operator for orchestrating Elasticsearch, Kibana, APM
    Server, Enterprise Search, and Beats.'
  url: https://hub.helm.sh/charts/elastic/eck-operator
  version: 1.4.1
- app_version: 3.9.0
  description: Kubernetes Operator based on the Operator SDK for creating and managing
    Grafana instances and dashboards.
  url: https://hub.helm.sh/charts/bitnami/grafana-operator
  version: 0.6.0
- app_version: v0.5.0
  description: Pipelinewise Operator for Kubernetes
  url: https://hub.helm.sh/charts/pipelinewise-operator/pipelinewise-operator
  version: 0.5.1
- app_version: v1.1.1
  description: A Helm chart for Kubernetes to deploy telegraf-operator
  url: https://hub.helm.sh/charts/influxdata/telegraf-operator
  version: 1.1.5
- app_version: 0.5.1
  description: Webhook Relay Operator provides an easy way to receive webhooks to
    an internal Kubernetes cluster without configuring public IP or load balancer
  url: https://hub.helm.sh/charts/webhookrelay-operator/webhookrelay-operator
  version: 0.3.1
- app_version: 6.1.17+0.gec409fd7
  description: A Helm chart for cf-operator, the k8s operator for deploying BOSH releases
  url: https://hub.helm.sh/charts/quarks/cf-operator
  version: 6.1.17+0.gec409fd7
- app_version: 2.1.0
  description: A Helm chart to deploy the Couchbase Autonomous Operator for easily
    deploying, managing, and maintaining Couchbase Clusters. Couchbase Server is a
    NoSQL document database with a distributed architecture for performance, scalability,
    and availability. It enables developers to build applications easier and faster
    by leveraging the power of SQL with the flexibility of JSON.
  url: https://hub.helm.sh/charts/couchbase/couchbase-operator
  version: 2.1.13
- app_version: 0.6.1
  description: CoreOS etcd-operator Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/banzaicloud-stable/etcd-operator
  version: 0.1.1
- app_version: "1.0"
  description: A Helm chart for flink on Kubernetes operator
  url: https://hub.helm.sh/charts/flink-operator/flink-operator
  version: 0.1.1
- app_version: master
  description: Grafana-operator Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/banzaicloud-stable/grafana-operator
  version: 0.3.0
- app_version: 1.9.1
  description: Helm chart for the Istio Operator.
  url: https://hub.helm.sh/charts/stevehipwell/istio-operator
  version: 2.1.3
- app_version: ""
  description: Helm chart for deploying Istio operator. This is a carbon-copy of the
    chart found in https://github.com/istio/istio, with container image versions pinned
  url: https://hub.helm.sh/charts/wiremind/istio-operator
  version: 1.9.1
- app_version: ""
  description: Helm chart for deploying Istio operator
  url: https://hub.helm.sh/charts/particuleio/istio-operator
  version: 1.7.0
- app_version: 0.4.4
  description: Nats operator creates/configures/manages nats clusters atop Kubernetes
  url: https://hub.helm.sh/charts/geek-cookbook/nats-operator
  version: 0.1.3
- app_version: 0.7.4
  description: NATS operator creates/configures/manages nats clusters atop Kubernetes
  url: https://hub.helm.sh/charts/nats/nats-operator
  version: 0.7.4
- app_version: v0.3.3
  description: A Helm chart for netris-operator
  url: https://hub.helm.sh/charts/netrisai/netris-operator
  version: 0.2.5
- app_version: 1.6.1
  description: Postgres Operator creates and manages PostgreSQL clusters running in
    Kubernetes
  url: https://hub.helm.sh/charts/wener/postgres-operator
  version: 1.6.1
- app_version: v0.2.8
  description: The Solr Operator enables easy management of Solr resources within
    Kubernetes.
  url: https://hub.helm.sh/charts/apache-solr/solr-operator
  version: 0.2.8
- app_version: 21.01.1
  description: A Helm chart for deploying NetApp's Trident CSI storage provisioner
    using the Trident Operator.
  url: https://hub.helm.sh/charts/netapp-trident/trident-operator
  version: 21.1.1
- app_version: v0.3.0
  description: Vault Operator by AppsCode - HashiCorp Vault Operator for Kubernetes
  url: https://hub.helm.sh/charts/appscode/vault-operator
  version: 0.3.0
- app_version: 0.2.8-rc0
  description: Pravega Zookeeper-operator Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/banzaicloud-stable/zookeeper-operator
  version: 0.3.0
- app_version: 0.13.0
  description: Helm chart to deploy [altinity-clickhouse-operator](https://github.com/Altinity/clickhouse-operator).  The
    ClickHouse Operator creates, configures and manages ClickHouse clusters running
    on Kubernetes.  Refer to operator repo for additional information.
  url: https://hub.helm.sh/charts/slamdev/altinity-clickhouse-operator
  version: 0.0.4
- app_version: 0.0.6
  description: Helm chart to deploy [external-secrets-operator](https://github.com/slamdev/external-secrets-operator).  External
    Secrets Operator watches changes in external key-value backends to sync them with
    Kubernetes Secrets and ConfigMaps.  Refer to operator repo for additional information.
  url: https://hub.helm.sh/charts/slamdev/external-secrets-operator
  version: 0.0.13
- app_version: 0.45.0
  description: Stripped down version of prometheus-operator to only provision the
    operator and nothing else
  url: https://hub.helm.sh/charts/banzaicloud-stable/prometheus-operator-standalone
  version: 13.2.1
- app_version: 0.1.11
  description: sops secrets operator
  url: https://hub.helm.sh/charts/sops-secrets-operator/sops-secrets-operator
  version: 0.6.6
- app_version: 1.0.0-beta.1
  description: A Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/astarte/astarte-operator
  version: 1.0.0-beta.1
- app_version: 1.0.0
  description: A Backup Operator Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/grdscloud/backup-operator
  version: 0.1.0
- app_version: v1.5.5
  description: backyards-operator manages Backyards on Kubernetes
  url: https://hub.helm.sh/charts/banzaicloud-stable/backyards-operator
  version: 1.5.5
- app_version: 0.1.14
  description: canary-operator manages canary deployments
  url: https://hub.helm.sh/charts/banzaicloud-stable/canary-operator
  version: 0.1.16
- app_version: 1.5.0
  description: 'Kubernetes operator which handles the provisioning and management
    of Apache Cassandra clusters. '
  url: https://hub.helm.sh/charts/k8ssandra/cass-operator
  version: 0.29.0
- app_version: 0.4.1
  description: A Helm chart for dast-operator
  url: https://hub.helm.sh/charts/banzaicloud-stable/dast-operator
  version: 0.3.0
- app_version: 0.5.0
  description: Datadog Operator
  url: https://hub.helm.sh/charts/datadog/datadog-operator
  version: 0.5.3
- app_version: 0.3.1
  description: Datadog Operator
  url: https://hub.helm.sh/charts/datadog-test/datadog-operator
  version: 0.1.2
- app_version: 0.2.3
  description: A Helm chart deploying the Ditto operator
  url: https://hub.helm.sh/charts/helm-charts/ditto-operator
  version: 0.1.10
- app_version: 0.1.0
  description: The Dynatrace Operator Helm chart for Kubernetes and OpenShift
  url: https://hub.helm.sh/charts/dynatrace/dynatrace-operator
  version: 0.1.0
- app_version: 0.7.0
  description: CoreOS etcd-operator Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/hkube/etcd-operator
  version: 0.9.7
- app_version: "1.8"
  description: Simplifying FPGA management in Kubernetes
  url: https://hub.helm.sh/charts/inaccel/fpga-operator
  version: 1.2.2
- app_version: v0.1.0
  description: Grafana Operator by AppsCode
  url: https://hub.helm.sh/charts/appscode/grafana-operator
  version: 0.1.0
- app_version: 0.1.4
  description: A Helm chart deploying the hawkBit operator
  url: https://hub.helm.sh/charts/helm-charts/hawkbit-operator
  version: 0.1.4
- app_version: 4.5.2
  description: Operator for the HiveMQ MQTT broker
  url: https://hub.helm.sh/charts/hivemq/hivemq-operator
  version: 0.8.3
- app_version: "1.0"
  description: inlets-operator for Kubernetes
  url: https://hub.helm.sh/charts/inlets/inlets-operator
  version: 0.12.0
- app_version: 0.9.1
  description: istio-operator manages Istio deployments on Kubernetes
  url: https://hub.helm.sh/charts/banzaicloud-stable/istio-operator
  version: 0.0.79
- app_version: 1.21.2
  description: jaeger-operator Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/jaegertracing/jaeger-operator
  version: 2.19.1
- app_version: 0.5.0
  description: Kubernetes native operator which fully manages Jenkins on Kubernetes
  url: https://hub.helm.sh/charts/jenkins/jenkins-operator
  version: 0.4.3
- app_version: v0.15.1
  description: kafka-operator manages Kafka deployments on Kubernetes
  url: https://hub.helm.sh/charts/banzaicloud-stable/kafka-operator
  version: 0.4.6
- app_version: 0.50.0
  description: Kanister-operator Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/kanister/kanister-operator
  version: 0.50.0
- app_version: v0.1.0
  description: Kubepack Operator by AppsCode
  url: https://hub.helm.sh/charts/appscode/kubepack-operator
  version: 0.1.0
- app_version: 3.9.1
  description: A Helm chart to install Banzai Cloud logging-operator
  url: https://hub.helm.sh/charts/banzaicloud-stable/logging-operator
  version: 3.9.2
- app_version: 0.1.0
  description: 'Installs and configures the Medusa Operator for managing Apache Cassandra
    backups '
  url: https://hub.helm.sh/charts/k8ssandra/medusa-operator
  version: 0.28.0
- app_version: v0.5.0-rc.2
  description: A Helm chart for mysql operator
  url: https://hub.helm.sh/charts/presslabs/mysql-operator
  version: 0.5.0-rc.2
- app_version: 0.2.0
  description: A Helm chart for deploying the Oracle MySQL Operator
  url: https://hub.helm.sh/charts/banzaicloud-stable/mysql-operator
  version: 0.1.0
- app_version: 1.0.0
  description: A MySQL Database Operator Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/grdscloud/mysql-operator
  version: 0.1.0
- app_version: 0.3.4
  description: K8S Operator for New Relic Services
  url: https://hub.helm.sh/charts/sstarcher/newrelic-operator
  version: 0.3.4+f2ca803
- app_version: 1.0.20
  description: A chart for deploying the OpenUnison operator
  url: https://hub.helm.sh/charts/tremolo/openunison-operator
  version: 2.0.2
- app_version: 1.6.0
  description: A Helm chart for Deploying the Percona Kubernetes Operator for Percona
    Server for MongoDB
  url: https://hub.helm.sh/charts/percona/psmdb-operator
  version: 0.1.1
- app_version: 0.0.6
  description: A Helm chart for pvc-operator
  url: https://hub.helm.sh/charts/banzaicloud-stable/pvc-operator
  version: 0.0.11
- app_version: 1.7.0
  description: A Helm chart for Deploying the Percona XtraDB Cluster Operator Kubernetes
  url: https://hub.helm.sh/charts/percona/pxc-operator
  version: 0.1.11
- app_version: 0.1.0
  description: 'Configures and installs the Reaper Operator for Apache Cassandra.
    This tool  manages an external repair process for Apache Cassandra clusters. '
  url: https://hub.helm.sh/charts/k8ssandra/reaper-operator
  version: 0.30.0
- app_version: v0.7.0
  description: A Helm chart for SOPS Operator
  url: https://hub.helm.sh/charts/craftypath/sops-operator
  version: 0.7.2
- app_version: v1beta2-1.2.0-3.0.0
  description: A Helm chart for Spark on Kubernetes operator
  url: https://hub.helm.sh/charts/spot/spark-operator
  version: 1.0.5
- app_version: v2.3.1
  description: Cloud Native storage for containers
  url: https://hub.helm.sh/charts/storageos/storageos-operator
  version: 0.4.4
- app_version: 1.11.2
  description: A Helm chart for banzaicloud/bank-vaults Vault operator
  url: https://hub.helm.sh/charts/banzaicloud-stable/vault-operator
  version: 1.11.3
- app_version: 0.11.0-alpha.0
  description: Presslabs WordPress Operator Helm Chart
  url: https://hub.helm.sh/charts/presslabs/wordpress-operator
  version: 0.11.0-alpha.0
- app_version: 1.2.4
  description: Provides easy installation of the Armory Spinnaker Operator
  url: https://hub.helm.sh/charts/armory/armory-spinnaker-operator
  version: 1.2.4
- app_version: 1.4.1
  description: 'A Helm chart for installing the ECK operator Custom Resource Definitions
    (CRD). '
  url: https://hub.helm.sh/charts/elastic/eck-operator-crds
  version: 1.4.1
- app_version: "1.0"
  description: 'A Helm chart for deploying External-Service-Operator which is repsonsible
    for creating and monitoring Services running outside of the cluster. The External-Service-Operator
    will create K8s Ressources for those Services, so they can be used, as they would
    just live inside the cluster. One of the Usecases can also be, that you want to
    expose an external Service via your Kubernetes Ingress Controller in order to
    have one central reverse Proxy. '
  url: https://hub.helm.sh/charts/crowdfox/external-service-operator
  version: 0.0.1
- app_version: ""
  description: Logging operator CR for Fluentd and Fluent-bit.
  url: https://hub.helm.sh/charts/banzaicloud-stable/logging-operator-fluent
  version: 1.0.1
- app_version: 1.6.1
  description: Postgres Operator UI provides a graphical interface for a convenient
    database-as-a-service user experience
  url: https://hub.helm.sh/charts/wener/postgres-operator-ui
  version: 1.6.1
- app_version: 0.46.0
  description: A Helm chart for Prometheus Operator Mixin
  url: https://hub.helm.sh/charts/portefaix-hub/prometheus-operator-mixin
  version: 0.1.0
- app_version: 0.0.2
  description: Stardog User and Role Operator
  url: https://hub.helm.sh/charts/appuio/stardog-userrole-operator
  version: 0.0.3
- app_version: ""
  description: A Helm chart for vald-helm-operator
  url: https://hub.helm.sh/charts/vald/vald-helm-operator
  version: 1.0.3
- app_version: v0.8.6
  description: A Helm chart for the github action runner operator.
  url: https://hub.helm.sh/charts/evryfs-oss/github-actions-runner-operator
  version: 2.5.10
- app_version: 0.9.5
  description: The Dynatrace OneAgent Operator Helm chart for Kubernetes and Openshift
  url: https://hub.helm.sh/charts/dynatrace/dynatrace-oneagent-operator
  version: 0.9.5
- app_version: 0.1.3-1
  description: A Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/pnnl-miscscripts/gitlab-runner-operator
  version: 0.1.5
- app_version: v0.1.0
  description: An operator for gitlab runners
  url: https://hub.helm.sh/charts/alekc/gitlab-runner-operator
  version: 0.1.0
- app_version: 1.8.2
  description: A Helm chart for MongoDB Enterprise Kubernetes Operator
  url: https://hub.helm.sh/charts/keyporttech/helm-mongodb-operator
  version: 0.1.0
- app_version: 0.1.0
  description: istio-release-operator manages canary deployments
  url: https://hub.helm.sh/charts/banzaicloud-stable/istio-release-operator
  version: 0.1.1
- app_version: 3.9.1
  description: A Helm chart to configure logging resource for the Logging operator
  url: https://hub.helm.sh/charts/banzaicloud-stable/logging-operator-logging
  version: 3.9.2
- app_version: 0.2.3
  description: Operator for managing NATS Streaming clusters running on Kubernetes.
  url: https://hub.helm.sh/charts/geek-cookbook/nats-streaming-operator
  version: 1.0.0
- app_version: 0.1.1
  description: Banzai K8s Nodepool Labels Operator
  url: https://hub.helm.sh/charts/banzaicloud-stable/nodepool-labels-operator
  version: 0.1.1
- app_version: v0.45.0
  description: Prometheus Operator CRDs
  url: https://hub.helm.sh/charts/appscode/prometheus-operator-crds
  version: 0.45.0
- app_version: v0.2.0
  description: Service level operator abstracts and automates the service level of
    Kubernetes applications by generation SLI & SLOs to be consumed easily by dashboards
    and alerts and allow that the SLI/SLO's live with the application flow.
  url: https://hub.helm.sh/charts/cloudposse/service-level-operator
  version: 0.1.0
- app_version: v0.3.0
  description: Service level operator abstracts and automates the service level of
    Kubernetes applications by generation SLI & SLOs to be consumed easily by dashboards
    and alerts and allow that the SLI/SLO's live with the application flow.
  url: https://hub.helm.sh/charts/softonic/service-level-operator
  version: 0.1.1
- app_version: 0.9.1
  description: Victoria Metrics Operator
  url: https://hub.helm.sh/charts/victoriametrics/victoria-metrics-operator
  version: 0.1.8
- app_version: v1.0.1
  description: Redis Database Assignment Operator
  url: https://hub.helm.sh/charts/wyrihaximusnet/redis-db-assignment-operator
  version: 1.0.5
- app_version: v1.2.9
  description: A Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/datawire/ambassador-operator
  version: 0.2.0
- app_version: 0.0.1
  description: Chart to configure and install Devtron. Devtron is a Kubernetes Orchestration
    system.
  url: https://hub.helm.sh/charts/devtron/devtron-operator
  version: 0.10.3
- app_version: ""
  description: A Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/banzaicloud-stable/hpa-operator
  version: 0.3.0
- app_version: ""
  description: Provides easy monitoring definitions for Kubernetes services, and deployment
    and management of Prometheus instances.
  url: https://hub.helm.sh/charts/cloudposse/prometheus-operator
  version: 0.2.0
- app_version: 0.1.2
  description: A Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/banzaicloud-stable/thanos-operator
  version: 0.1.1
- app_version: 0.2.2
  description: A Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/spot/wave-operator
  version: 0.2.2
- app_version: v0.4.0
  description: A Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/banzaicloud-stable/integrated-service-operator
  version: 0.4.0
- app_version: 1.0.7
  description: ""
  url: https://hub.helm.sh/charts/softonic/rate-limit-operator
  version: 1.0.7
- app_version: ""
  description: Seldon Core CRD and controller helm chart for Kubernetes.
  url: https://hub.helm.sh/charts/seldon/seldon-core-operator
  version: 1.6.0
- app_version: 0.1.9-1
  description: A Helm chart for Kubernetes
  url: https://hub.helm.sh/charts/pnnl-miscscripts/tenant-namespace-operator
  version: 0.1.11
- app_version: 0.5.1
  description: Kubernetes-Operator to simplify RBAC configurations
  url: https://hub.helm.sh/charts/ckotzbauer/access-manager
  version: 0.5.1
- app_version: v1.0.3
  description: Kubernetes and OpenShift Backup Operator based on restic
  url: https://hub.helm.sh/charts/appuio/k8up
  version: 1.0.3
- app_version: 0.46.0
  description: kube-prometheus collects Kubernetes manifests to provide easy to operate
    end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus
    Operator.
  url: https://hub.helm.sh/charts/wener/kube-prometheus
  version: 4.1.2
- app_version: 0.46.0
  description: kube-prometheus collects Kubernetes manifests to provide easy to operate
    end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus
    Operator.
  url: https://hub.helm.sh/charts/bitnami/kube-prometheus
  version: 4.1.2
- app_version: 0.46.0
  description: kube-prometheus-stack collects Kubernetes manifests, Grafana dashboards,
    and Prometheus rules combined with documentation and scripts to provide easy to
    operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus
    Operator.
  url: https://hub.helm.sh/charts/prometheus-community/kube-prometheus-stack
  version: 14.0.1
- app_version: 0.44.0
  description: kube-prometheus-stack collects Kubernetes manifests, Grafana dashboards,
    and Prometheus rules combined with documentation and scripts to provide easy to
    operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus
    Operator.
  url: https://hub.helm.sh/charts/prometheus-worawutchan/kube-prometheus-stack
  version: 12.8.0
- app_version: v2.4.0
  description: An operator for synthetic test and monitoring. Works great with Prometheus.
  url: https://hub.helm.sh/charts/kuberhealthy/kuberhealthy
  version: 53.0.0
- app_version: 7.2.1+0.gaeb6ef3
  description: A Helm chart for quarks-operator, the k8s operator for deploying BOSH
    releases
  url: https://hub.helm.sh/charts/quarks/quarks
  version: 7.2.1+0.gaeb6ef3
- app_version: ""
  description: 'Creates a CassandraBackup custom resource instance. This chart should
    be used in conjunction with the Medusa Operator chart. '
  url: https://hub.helm.sh/charts/k8ssandra/backup
  version: 0.26.0
- app_version: 0.13.0-rc.98.g1eb0776
  description: Crossplane - Managed Cloud Resources Operator
  url: https://hub.helm.sh/charts/crossplane/crossplane-controllers
  version: 0.13.0-rc.98.g1eb0776
- app_version: 0.13.0-rc.98.g1eb0776
  description: Crossplane - Managed Cloud Resources Operator
  url: https://hub.helm.sh/charts/crossplane/crossplane-types
  version: 0.13.0-rc.98.g1eb0776
- app_version: 1.6.4
  description: Flagger is a progressive delivery operator for Kubernetes
  url: https://hub.helm.sh/charts/flagger/flagger
  version: 1.6.4
- app_version: v0.3.1
  description: Obtains the resource quotas from a GCP project and allows to export
    them to Prometheus
  url: https://hub.helm.sh/charts/softonic/gcp-quota-exporter
  version: 2.0.0
- app_version: "1.0"
  description: An Operator that installs Helm Charts
  url: https://hub.helm.sh/charts/kubedex/helm-controller
  version: 0.4.0
- app_version: v0.7.6
  description: A Helm chart for Kubernetes Global Balancer
  url: https://hub.helm.sh/charts/k8gb/k8gb
  version: 0.7.6
- app_version: 0.1.1
  description: Kubernetes operator which update AWS IAM TrustPolicies.
  url: https://hub.helm.sh/charts/k8s-aws-iam-controller/k8s-aws-iam-controller
  version: 0.1.1
- app_version: v0.2.0
  description: 'KubePreset: Streamline Application Connectivity with Backing Services'
  url: https://hub.helm.sh/charts/kubepreset/kubepreset
  version: 0.2.0
- app_version: 0.38.1
  description: Provides easy monitoring definitions for Kubernetes services, and deployment
    and management of Prometheus instances.
  url: https://hub.helm.sh/charts/choerodon/kube-prometheus
  version: 9.3.1
- app_version: "1.0"
  description: A Helm chart for easy deployment of a MySQL cluster with MySQL operator.
  url: https://hub.helm.sh/charts/presslabs/mysql-cluster
  version: 0.3.0
- app_version: 2.5.0
  description: A Demo application for the logging-operator AWS CloudWatch backend
  url: https://hub.helm.sh/charts/banzaicloud-stable/nginx-logging-cw-demo
  version: 2.5.0
- app_version: 2.5.0
  description: A Demo application for the logging-operator
  url: https://hub.helm.sh/charts/banzaicloud-stable/nginx-logging-demo
  version: 2.5.0
- app_version: 2.5.0
  description: A Demo application for the logging-operator
  url: https://hub.helm.sh/charts/banzaicloud-stable/nginx-logging-es-demo
  version: 2.5.0
- app_version: 2.5.0
  description: A Demo application for the logging-operator
  url: https://hub.helm.sh/charts/banzaicloud-stable/nginx-logging-loki-demo
  version: 2.5.0
- app_version: ""
  description: Prometheus instance created by the CoreOS Prometheus Operator
  url: https://hub.helm.sh/charts/cloudposse/prometheus
  version: 0.2.1
- app_version: 1.6.0
  description: A Helm chart for installing Percona Server MongoDB Cluster Databases
    using the PSMDB Operator.
  url: https://hub.helm.sh/charts/percona/psmdb-db
  version: 0.1.2
- app_version: 1.7.0
  description: A Helm chart for installing Percona XtraDB Cluster Databases using
    the PXC Operator.
  url: https://hub.helm.sh/charts/percona/pxc-db
  version: 0.1.16
- app_version: 1.0.218
  description: A Helm chart for quarks-job, a k8s operator for quarks jobs
  url: https://hub.helm.sh/charts/quarks/quarks-job
  version: 1.0.218
- app_version: 1.0.758
  description: A Helm chart for quarks-secret, a k8s operator to create secrets
  url: https://hub.helm.sh/charts/quarks/quarks-secret
  version: 1.0.758
- app_version: 0.10.0
  description: A Kubernetes operator that simplifies the management of Role Bindings
    and Service Accounts.
  url: https://hub.helm.sh/charts/fairwinds-stable/rbac-manager
  version: 1.8.1
- app_version: 0.1.3
  description: Kubernetes operator which consolidates custom resources into `aws-auth`
    ConfigMap.
  url: https://hub.helm.sh/charts/aws-eks-iam-auth-controller/rustrial-aws-eks-iam-auth-controller
  version: 0.1.3
- app_version: 0.0.3
  description: Kubernetes operator for GitOps SealedSecrets.
  url: https://hub.helm.sh/charts/k8s-gitops-secrets/rustrial-k8s-gitops-secrets-controller
  version: 0.0.3
- app_version: 1.1.1
  description: An operator to take scheduled snapshots of Kubernetes persistent volumes
  url: https://hub.helm.sh/charts/backube-helm-charts/snapscheduler
  version: 1.2.1
- app_version: ""
  description: Install and configure Terraform Cloud Operator on Kubernetes.
  url: https://hub.helm.sh/charts/hashicorp/terraform
  version: 1.0.0
- app_version: v0.0.17
  description: Terraform controller Helm Chart
  url: https://hub.helm.sh/charts/terraform-controller/terraform-controller
  version: 0.0.17
- app_version: ""
  description: A Helm chart to test the secureCodeBox operator
  url: https://hub.helm.sh/charts/seccurecodebox/test-scan
  version: 2.5.2
```

Reply:
Ok sounds good. I was hoping of combining the "this is an operator for Redpanda" part with what is Redpanda.

Reply:
The github link and home page is enough to describe what is the Redpanda.